style.css

```diff
@@ -0,0 +1,48 @@
+html {
+    background-color: grey
+    }
+    .title-text {
+    text-align: center;
+    color: white;}
+    
+    .button {
+    color: #FFFFFF;
+    text-decoration: none; 
+     background: #000000;
+     padding: .50em 4.5em;
+     border: 4px solid white;
+     pisitive:relative;
+    }
+    
+    .button:hover {
+    background: #FF0000;
+    }
+    .button:active{
+      top:1px;
+      }
+    
+    html, body {
+      width: 100%;
+      height: 100%;
+      margin: 0;
+    }
+    
+    .inner {
+      position: relative;
+      top: 50%;
+      transform: translate(0, -50%)
+    }
+    
+    body {
```

Comment:
background added to body, should be added only to the top element (e.g. hero (search for hero web page))